src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisEvent.scala

```diff
@@ -23,7 +21,7 @@ case class Analysis(
   locations: List[Location] = List(),
   sentiments: List[Double] = List(),
   moods: List[Tag] = List(),
-  genders: List[Tag] = List(),
+  //genders: List[Tag] = List(),
```

Comment:
Commenting out as this isnt quite ready for primetime just yet. This will be going out within the next two weeks.